test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -18,23 +18,30 @@ public void isAnyNull() {
         assertFalse(Utils.isAnyNull(new Object(), new Object()));
         assertFalse(Utils.isAnyNull("test"));
         assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(123));
+        assertFalse(Utils.isAnyNull(false));
 
```

Comment:
@LiHaoTan  Ya, test<MethodName> is a must have part, where others are optional. Personally I would like to split the testing to every sub state because if a sub state fails, it is very clear that which part of the code is buggy. If we put everything together, we would need to manually find where the bug is, thus deceit the purpose of unit testing. :) But, there are many different cases also, I can't really tell you which way of doing is the best, you would need to understand the practice itself. ^^